lib/middleware.js

```diff
@@ -115,10 +114,24 @@ module.exports = function getMiddleware(watcher, options) {
       var context = {
         stack: buildError.stack,
         liveReloadPath: options.liveReloadPath,
-        payload: buildError.broccoliPayload
-      }
+        payload: {
+          stack: buildError.stack,
+          message: buildError.message,
+          pos: buildError.pos,
+          loc: buildError.loc,
+          _babel: buildError._babel,
+          codeFrame: buildError.codeFrame,
+          file: buildError.file,
+          extension: path.extname(buildError.file).replace('.', ''),
+          treeDir: buildError.treeDir,
+        }
+      };
+
       response.setHeader('Content-Type', 'text/html')
       response.writeHead(500)
+
+var errorTemplate = handlebars.compile(fs.readFileSync(path.resolve(__dirname, 'templates/error.html')).toString())
```

Comment:
hack, so I could iterate on the style without having to reload the server all the time.